noxfile.py

```diff
@@ -87,7 +88,24 @@ def test(session: nox.Session) -> None:
     if _should_install(session):
         session.install("-e", ".[test]")
     session.run(*split("pybabel compile -d src/pydata_sphinx_theme/locale -D sphinx"))
-    session.run("pytest", *session.posargs)
+    session.run("pytest", "-m", "not a11y", *session.posargs)
+
+
+@nox.session()
+def a11y(session: nox.Session) -> None:
+    """Run the accessibility test suite only."""
+    if _should_install(session):
+        session.install("-e", ".[test, a11y]")
+        # Install the drivers that Playwright needs to control the browsers.
+        if not (os.environ.get("CI") or os.environ.get("GITPOD_WORKSPACE_ID")):
```

Comment:
I decided to remove the `playwright install` for CI from the `noxfile` since I think it is better to delegate this to the CI directly and leverage cache there

Also, I was not sure about leaving the `GITPOD_WORKSPACE_ID` bit. I am a Gitpod user, but I wonder if it will cause confusion among the rest of folks who do not use Gitpod